synapse/config/server.py

```diff
@@ -695,9 +696,15 @@ def generate_config_section(
         # sign up in a short space of time never to return after their initial
         # session.
         #
+        # 'mau_suppress_alerting' is a means of limiting client side alerting
+        # should the mau limit be reached. This is useful for small instances
+        # where the admin has 5 mau seats (say) for 5 specific people and no
+        # interest increasing the mau limit further. Defaults to False.
+        #
         #limit_usage_by_mau: False
         #max_mau_value: 50
         #mau_trial_days: 2
+        #mau_suppress_alerting: True
```

Comment:
It might make more intuitive sense to have the option be to alert, with it defaulting to True?

Reply:
Agreed, iirc lately we've mainly been trying to add options to enable stuff (which can default to `True`) rather than ones to disable stuff to avoid confusion.

Reply:
My thinking was that almost all cases alerting will be the default so the option made sense to be in the negative. Sounds like it is confusing, so I have inverted.